test/frontend/OfficerDB.test.js

```diff
@@ -10,8 +10,8 @@ import { Table } from 'reactstrap'
 
 Enzyme.configure({ adapter: new Adapter() })
 
-describe('<Table />', () => {
-  it('Should render a <Table /> component with one child', () => {
+describe('<OfficerManager />', () => {
+  it('Should render a <OfficerManager /> component with one child', () => {
```

Comment:
Keep this line as `Table`, since that is what we are testing.
```suggestion
  it('Should render a <Table /> component with one child', () => {
```